contracts/Unibright.sol

```diff
@@ -118,14 +125,17 @@ contract UBTSplitter is Context, Ownable {
         string memory destinationAddress
     ) public zeroAddress(token) emptyString(destinationAddress) {
         require(
-            whitelistedTokens[address(token)],
```

Comment:
we should remove token from function arguments since we will only use one from constructor, same for other functions